backstop/tests/carousel.html

```diff
@@ -0,0 +1,471 @@
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>Carousel | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      section[id^='demo'] {
+        margin-bottom: 22px;
+      }
+
+      /* These dimensions equal 16.9 ratio */
+      .demo-carousel-width {
+        width: 600px;
+      }
+
+      .demo-carousel-height {
+        height: 337.5px;
+      }
+
+      /* Slide gradients */
+      [data-slide-id="0"] { background: linear-gradient(to right, #e65c00, #f9d423); text-align: center; vertical-align: middle; line-height: 337.5px; }
+      [data-slide-id="1"] { background: linear-gradient(to right, #2193b0, #6dd5ed); }
+      [data-slide-id="2"] { background: linear-gradient(to right, #cc2b5e, #753a88); }
+      [data-slide-id="3"] { background: linear-gradient(to right, #00467f, #a5cc82); }
+      [data-slide-id="4"] { background: linear-gradient(to right, #ec008c, #fc6767); }
+      [data-slide-id="5"] { background: linear-gradient(to right, #1488cc, #2b32b2); }
+      [data-slide-id="6"] { background: linear-gradient(to right, #bbd2c5, #536976); }
+      [data-slide-id="7"] { background: linear-gradient(to right, #9796f0, #fbc7d4); }
+      [data-slide-id="8"] { background: linear-gradient(to right, #b79891, #94716b); }
+      [data-slide-id="9"] { background: linear-gradient(to right, #acb6e5, #86fde8); }
+      [data-slide-id="10"] { background: linear-gradient(to right, #ffe000, #799f0c); }
+
+    </style>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/chevron-left"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/chevron-right"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/pause"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/play"
+    ></script>
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>Carousel</h1>
+    <hr />
+
+    <h2>Default</h2>
+    <section id="demo-default">
+
+      <div
+        class="iui-carousel demo-carousel-width"
+        aria-roledescription="carousel"
+        aria-label="Introduction to iTwinUI"
+        aria-live="off"
+        aria-atomic="true"
+      >
+        <nav class="iui-carousel-navigation">
+          <div class="iui-carousel-navigation-left">
+            <button
+              class="iui-button iui-borderless iui-small iui-carousel-navigation-animation-button"
+              type="button"
+              aria-label="Pause animation"
+              aria-pressed="true"
```

Comment:
I think you don't need aria-pressed here because you have aria-label.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed#:~:text=don't%20use%20aria-pressed

Reply:
I removed it